tests/crafting_test.cpp

```diff
@@ -854,12 +854,9 @@ TEST_CASE( "tools_use_charge_to_craft", "[crafting][charge]" )
             item soldering = tool_with_ammo( "soldering_iron_portable", 20 );
             REQUIRE( soldering.ammo_remaining() == 20 );
             tools.push_back( soldering );
-            item plastic_molding = tool_with_ammo( "vac_mold", 4 );
-            REQUIRE( plastic_molding.ammo_remaining() == 4 );
-            tools.push_back( plastic_molding );
 
             THEN( "crafting succeeds, and uses charges from each tool" ) {
-                prep_craft( recipe_carver_off, tools, true, 0, false, false );
+                prep_craft( recipe_carver_off, tools, true, 0, false, true );
```

Comment:
It's ok to plug in tools here because the tools we're interested in charges of are not pluggable. This just lets the recipe uses the vacuum molder.